src/ui/Errors/Loading/Loading.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { addDecorator, storiesOf } from '@storybook/react';
+import { withConsole } from '@storybook/addon-console';
+import { ErrorLoading } from 'ui/Errors';
+import Button from 'ui/Button/Button';
+
+storiesOf('ErrorLoading', module)
+  .addDecorator((storyFn, context) => withConsole()(storyFn)(context))
+  .add('default', () => <ErrorLoading mainText="Default Error Loading Text" />)
+  .add('attendees', () => <ErrorLoading mainText="The attendees list isn’t loading right now." />)
+  .add('with button', () => (
+    <ErrorLoading mainText="The attendees list isn’t loading right now.">
+      <Button
+        bordered
+        variant="primary"
+        uppercase={false}
+        height="42px"
+        width="168px"
+        style={{ marginTop: '26px' }}
```

Comment:
We try to avoid passing explicit styles into a component. There should just be a `ButtonContainer` div that has this margin on it wrapped around this.